style.css

```diff
@@ -546,8 +545,66 @@ button {
   transition: all 1s ease 0.5s;
 }
 
-.nav_wrapper {
+.header_wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 3rem;
+  align-content: center;
+
+  /* margin-left: 15px;
+  margin-right: 15px; */
+}
+
+.navigation {
   display: none;
+  position: fixed;
+  z-index: 1;
+  padding-top: 50px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(96, 112, 255);
+  background-color: rgb(96, 112, 255, 0.95);
+  mix-blend-mode: multiply;
+  backdrop-filter: blur(15px);
+}
+
+.btn-wrapper {
+  position: fixed;
+  top: 10px;
+  right: 10px;
+}
+
+.close-btn {
+  border: none;
+  cursor: pointer;
+  width: 45px;
+  outline: none;
+  background: none;
+}
+
+.nav_wrapper {
+  list-style: none;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 80%;
+  z-index: 100;
+}
+
+.nav_item {
+  padding: 50px 30px;
+}
```

Comment:
- [ ] Great job so far :muscle: However I noticed here that the padding you applied to the list items was too big making the design look slightly different from the Figma Design. So kindly try to fix this and try to stick to the Figma design as much as possible :wink:
- Figma Design
![image](https://user-images.githubusercontent.com/102699958/215598321-d858a8ef-c85e-4bb3-a1da-4f3db75e43d4.png)
- Your Design
![image](https://user-images.githubusercontent.com/102699958/215598418-67d0e45e-0325-480b-8d96-caf77d3d2150.png)

Reply:
worked on